src/index.js

```diff
@@ -0,0 +1,234 @@
+    count5++;
+  }
+}
+console.log("[pregunta #5]: ", count5); // Answer should be: 4
+
+// Question 6: Find the product of the numbers 1 to 5.
+let product6 = 1;
+for (let n = 1; n <= 5; n++) {
+  // Your code here
+  product6 *= n;
+}
+console.log("[pregunta #6]: ", product6); // Answer should be: 120
+
+// Question 7: Create a string from an array that tells the story of David and Goliath.
+let story7 = ["David", "fought", "Goliath", "and", "won."];
+let storyString7 = "";
+for (let o = 0; o < story7.length; o++) {
+  // Your code here
+  storyString7 += story7[o] + " ";
+}
+console.log("[pregunta #7]: ", storyString7); // Answer should be: David fought Goliath and won.
+
+// Question 8: Create an acronym from the phrase "In The Beginning".
+let phrase8 = "In The Beginning";
+let acronym8 = "";
+for (let p = 0; p < phrase8.length; p++) {
+  // Your code here
+  if (p === 0) {
+    acronym8 += phrase8[p];
+  }
+  if (phrase8[p] === " ") {
+    acronym8 += phrase8[p + 1];
+  }
+}
+console.log("[pregunta #8]: ", acronym8.toUpperCase()); // Answer should be: ITB
+
+// Question 9: Convert the string "Noah" to uppercase.
+let name9 = "Noah";
+let upperName9 = "";
+for (let q = 0; q < name9.length; q++) {
+  // Your code here
+  upperName9 = name9.toUpperCase();
+}
+console.log("[pregunta #9]: ", upperName9); // Answer should be: NOAH
+
+// Question 10: Find the length of the string "Revelation" without using the `.length` property.
+let name10 = "Revelation";
+let length10 = 0;
+for (let r = 0; name10[r] !== undefined; r++) {
+  // Your code here
+  length10++;
+}
+console.log("[pregunta #10]: ", length10); // Answer should be: 10
+
+// Additional 10 Questions on For-loops and Biblical Trivia
+
+// Question 11: Count the number of occurrences of the letter 'e' in the word "Jericho".
+let city11 = "Jericho";
+let count11 = 0;
+for (let s = 0; s < city11.length; s++) {
+  // Your code here
+  if (city11[s] === "e") {
+    count11++;
+  }
+}
+console.log("[pregunta #11]: ", count11); // Answer should be: 1
+
+// Question 12: Reverse the string "Bethlehem".
+let town12 = "Bethlehem";
+let reversedTown12 = "";
+for (let t = town12.length - 1; t >= 0; t--) {
+  // Your code here
+  reversedTown12 += town12[t];
+}
+console.log("[pregunta #12]: ", reversedTown12); // Answer should be: mehtelhteB
+
+// Question 13: Sum the integers from 1 to 40 (the number of days of Jesus' temptation).
+let sum13 = 0;
+for (let u = 1; u <= 40; u++) {
+  // Your code here
+  sum13 += u;
+}
+console.log("[pregunta #13]: ", sum13); // Answer should be: 820
+
+// Question 14: Concatenate the list of the Twelve Apostles into a single string separated by a comma.
+let apostles14 = [
+  "Peter",
+  "James",
+  "John",
+  "Andrew",
+  "Philip",
+  "Bartholomew",
+  "Matthew",
+  "Thomas",
+  "James",
+  "Thaddeus",
+  "Simon",
+  "Judas"
+];
+let apostlesString14 = "";
+for (let v = 0; v < apostles14.length; v++) {
+  // Your code here
+  apostlesString14 = apostles14.join(", ");
```

Comment:
Esta no es la manera de hacerlo en un for loop. Aqui estas haciendo el join  una vez por cada iteracion del for loop. Utiliza la concatenacion y checa que en el ultimo string no pongas coma.